### PR TITLE
Add code for drawing shapes and the player hand

### PR DESCRIPTION
**Pull request by ivansalazar**

### CODE REVIEW

File: src/qwircl/core.cljs [hunk 1]
```diff
@@ -43,13 +54,90 @@
...
+
 (defn draw-state [state]
+  (q/frame-rate 120)
   (q/background 240)
-  (q/fill 0 255 255)
-  (q/stroke 0 0 0)
-  (doseq [x (range tiles) 
-          y (range tiles)]
-    (q/rect (* size x) (* size y) size size)))
+  (apply q/fill background-color)
```
Comment (tclamb) [inline src/qwircl/core.cljs]: `q/fill` has a unary overload that takes an rgb vector. I think `set- color` uses `apply` too

File: src/qwircl/core.cljs [hunk 2]
```diff
@@ -43,13 +54,90 @@
...
+           :yellow [255 255 0]
+           :black [0 0 0]
+           ; [0 255 255]
+           background-color)))
+
+(defn draw-black-background [x y]
+  (set-color :black)
+  (q/rect (* size x) (* size y) size size))
+
+(defn draw-tile [x y tile]
```
Comment (tclamb) [inline src/qwircl/core.cljs:74]: I love this!

File: src/qwircl/core.cljs [hunk 3]
```diff
@@ -43,13 +54,90 @@
...
+      (if-let [tile (get-in grid [x y])]
+        (draw-tile x y tile)
+        (draw-empty-space x y)))))
+
+(defn draw-header [player]
+  (q/fill 0 0 0)
+  (q/text (:name player) 10 header)
+  (dotimes [x (count (:hand player))]
+    (draw-tile x 0 ((:hand player) x)))
+  (apply q/fill background-color))
```
Comment (tclamb) [inline src/qwircl/core.cljs]: I think you intended to replace these `q/fill`s with `set-color`

**Review by tclamb — APPROVED** [review]

File: src/qwircl/core.cljs (hunk 1 above)
Comment (ivansalazar) [reply src/qwircl/core.cljs]: Done.

File: src/qwircl/core.cljs (hunk 3 above)
Comment (ivansalazar) [reply src/qwircl/core.cljs]: Good catch.

**Pull request merged**